source/features/more-dropdown.tsx

```diff
@@ -59,9 +58,9 @@ async function init(): Promise<void> {
 		'.UnderlineNav-body'
 	].join());
 
-	const reference = getCurrentCommittish() ?? await getDefaultBranch();
-	const compareUrl = buildRepoURL('compare', reference);
-	const commitsUrl = buildRepoURL('commits', reference);
+	const reference = await getCurrentCommittishAnywhere();
```

Comment:
Default branch is still needed, I mistyped that
```suggestion
	const reference = await getCurrentCommittishAnywhere() ?? await getDefaultBranch();
```

Reply:
Why? I checked even without our feature I was unable to find a page that did not have it.  I ment to open an issue

Reply:
> Why? I checked even without our feature I was unable to find a page that did not have it. I ment to open an issue

Oh my I disabled the wrong feature and I thought GitHub added it LOL